guides/source/association_basics.md

```diff
@@ -788,7 +788,23 @@ module MyApplication
 end
 ```
 
-This will work fine, because both the `Supplier` and the `Account` class are defined within the same scope. But the following will _not_ work, because `Supplier` and `Account` are defined in different scopes:
+This will work fine, because both the `Supplier` and the `Account` class are defined within the same scope. This is highly beneficial as it allows you to organize your models according to their scope. For example:
+
+```ruby
+# app/models/business/account.rb
+class Business::Account < ApplicationRecord
+  belongs_to :business
+end
+
+# app/models/business.rb
+class Business < ApplicationRecord
+  has_many :accounts
+end
+```
+
+This organization allows you to structure your models into folders based on their scope. It is crucial to note that this does not affect the naming of your tables. For instance, if there is a `Business::Supplier` model, there must also be a `business_suppliers` table.
+
+But the following will _not_ work, because `Supplier` and `Account` are defined in different scopes:
```

Comment:
The "But" doesn't make much sense here anymore, as it no longer applies to the previous sentence: 
```suggestion
Note that he following will _not_ work, because `Supplier` and `Account` are defined in different scopes (`MyApplication::Business` and `MyApplication::Billing`):
```